.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
                 test/test.sh
               "
           elif [[ $OSTYPE == darwin* ]]; then
-            brew install bash # This should pull in the correct bash version
+            brew install bash coreutils # This should pull in the correct bash version
```

Comment:
Fix the comment. This is not just about bash anymore. What are we installing coreutils for?